server-kotlin/src/city_sync/CitySynchronizationService.kt

```diff
@@ -0,0 +1,220 @@
+package city_sync
```

Comment:
Missing `digital.cesko` package prefix. Idea does that to me as well.

Reply:
Fixed